.ci/run-opensearch.sh

```diff
@@ -65,15 +65,19 @@ END
   if [[ "$i" == "$((NUMBER_OF_NODES-1))" ]]; then local_detach=$DETACH; fi
 
   echo -e "\033[34;1mINFO: building $CLUSTER container\033[0m"
-  echo 'cluster is' $CLUSTER
   docker build \
     --file=.ci/$CLUSTER/Dockerfile \
-    --build-arg SECURE_INTEGRATION=false \
-    --tag=$CLUSTER \
+    --build-arg SECURE_INTEGRATION=$SECURE_INTEGRATION \
+    --tag=$CLUSTER-secure-$SECURE_INTEGRATION \
     .
 
   echo -e "\033[34;1mINFO:\033[0m Starting container $node_name \033[0m"
   set -x
+  healthcmd="curl -vvv -s --fail localhost:9200/_cluster/health || exit 1"
+  if [[ "$SECURE_INTEGRATION" == "true" ]]; then
```

Comment:
Could be simplified if you use it as a boolean expression:
```
if "$SECURE_INTEGRATION"; then
...
```

Reply:
Same as above